e2e/process-services-cloud/edit-task-filters-component.e2e.ts

```diff
@@ -34,16 +33,13 @@ describe('Edit task filters cloud', () => {
         const tasksCloudDemoPage = new TasksCloudDemoPage();
         let tasksService: TasksService;
 
-        let silentLogin;
-        const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP;
+        const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP.name;
         const completedTaskName = StringUtil.generateRandomString(), assignedTaskName = StringUtil.generateRandomString();
         let assignedTask;
 
-        beforeAll(async () => {
-            silentLogin = false;
-            settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, silentLogin);
+        beforeAll(async (done) => {
+            settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false);
```

Comment:
It's better to have a variable called 'silentLogin' set to false, because in this way we can understand from the test what kind of login is the test doing.